diff --git a/node_modules/@ai-sdk/react/dist/index.d.ts b/node_modules/@ai-sdk/react/dist/index.d.ts
index cb7c14c..d2a6374 100644
--- a/node_modules/@ai-sdk/react/dist/index.d.ts
+++ b/node_modules/@ai-sdk/react/dist/index.d.ts
@@ -62,9 +62,18 @@ type UseAssistantHelpers = {
 };
 declare function useAssistant({ api, threadId: threadIdParam, credentials, headers, body, onError, fetch, }: UseAssistantOptions): UseAssistantHelpers;
 
+
+export type UIMessageExtended = UIMessage & {
+    type?: string;
+    fileUrl?: string;
+    fileName?: string;
+    fileType?: string;
+}
+
+
 type UseChatHelpers = {
     /** Current messages in the chat */
-    messages: UIMessage[];
+    messages: UIMessageExtended[];
     /** The error object of the API request */
     error: undefined | Error;
     /**
